shared/data/constants.js

```diff
@@ -96,6 +96,18 @@ module.exports = {
             "format": "text",
             "source": "external"
         },
+        {
+            "name": "brokenCanvasSiteList",
+            "url": "/data/broken-canvas-sites.txt",
```

Comment:
@Charlie-belmer these need to be remote lists probably. Shall I add the empty files to the alowlist repo?

Reply:
Yeah that is the way to go - let me know if you need a hand with it.